docs/python/tutorials/queue-tutorial.md

```diff
@@ -200,6 +200,41 @@ def on_user_task_submission(user_id: str, task: Task):
         queue.enqueue(process_task, task)
 ```
 
+Sometimes, you want to apply global limits to a partitioned queue.
+You can do this with **multiple levels of queueing**.
+Create two queues: a partitioned queue with per-partition limits and a non-partitioned queue with global limits.
+Enqueue a "concurrency manager" workflow to the partitioned queue, which then enqueues your actual workflow
+to the non-partitioned queue and awaits its result.
+This ensures both queues' flow control limits are enforced on your workflow.
+For example:
+
+```python
+# By using two levels of queueing, we enforce both a concurrency limit of 1 on each partition
+# and a global concurrency limit of 5, meaning that no more than 5 tasks can run concurrently
+# across all partitions (and at most one task per partition).
+concurrency_queue = Queue("concurrency-queue", concurrency=5)
+partitioned_queue = Queue("partitioned-queue", partition_queue=True, concurrency=1)
+
+@app.get("/queue")
+@DBOS.workflow()
+def on_user_task_submission(user_id: str, task: Task):
+    # First, enqueue a "concurrency manager" workflow to the partitioned
+    # queue to enforce per-partition limits.
+    with SetEnqueueOptions(queue_partition_key=user_id):
+        partitioned_queue.enqueue(process_task, task)
+
+@DBOS.workflow()
+def concurrency_manager(task):
+    # The "concurrency manager" workflow enqueues the process_task
+    # workflow on the non-partitioned queue and awaits its results
+    # to enforce global flow control limits.
+    return concurrency_queue.enqueue(process_task, task).get_result()
+
+@DBOS.workflow()
+def process_task(task):
+    ...
+```
+
```

Comment:
The handler should first enqueue `concurrency_manager` in the `concurrency_queue` and that workflow should dispatch the task in a `partitioned_queue` partition. This code is dispatching the task straight to `partitioned_queue`

Reply:
I see, I copy-pasted it wrong haha--I do have a working example